rele/config.py

```diff
@@ -36,7 +38,7 @@ def encoder(self):
         return getattr(module, class_name)
 
 
-def setup(setting, **kwargs):
+def setup(setting={}, **kwargs):
```

Comment:
A dict as default value for an argument is bad because the reference keeps being modified by subsequent calls.

Reply:
I see, so best to set it to `None` and then later set it to an empty dict if it is None?

Reply:
Yep!